src/main/java/com/ioet/bpm/skills/domain/converters/CategoryConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.ioet.bpm.skills.domain.converters;
+
+
+import com.amazonaws.services.dynamodbv2.datamodeling.DynamoDBTypeConverter;
+import com.ioet.bpm.skills.domain.Category;
```

Comment:
Why do we need a converter?

Reply:
I was wondering the same. 
It is because I we want to use Arbitrary Data we need to define converters to provide a way to map our complex object (Category) to values supported by DynamoDB.
Here more details https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBMapper.DataTypes.html